src/Components/ProgressiveOnboarding/ProgressiveOnboardingFollowArtist.tsx

```diff
@@ -32,7 +34,10 @@ export const __ProgressiveOnboardingFollowArtist__: FC<ProgressiveOnboardingFoll
     counts.isReady &&
     counts.followedArtists === 0 &&
     // If you've already dismissed the alerts onboarding OR you're not on the artist page.
```

Comment:
Can you update this comment to reflect the "OR you're logged out"

Reply:
done, I took into consideration that we never dismiss the `alerts onboarding` if you're being onboarded but that's not 100% from the new things we're doing with progressive onboarding and logged out status 👍